lib/issue_templates_issues_hook.rb

```diff
@@ -6,30 +6,38 @@ class IssueTemplatesIssuesHook < Redmine::Hook::ViewListener
   
   def view_layouts_base_html_head(context = {})
     o = stylesheet_link_tag('issue_templates', :plugin => 'redmine_issue_templates')
-    if (context[:controller].class.name == 'IssuesController' and 
-      context[:controller].action_name != 'index') or 
-      (context[:controller].class.name == 'IssueTemplatesController') or
+    if (context[:controller].class.name == 'IssuesController' &&
+        context[:controller].action_name != 'index') ||
+      (context[:controller].class.name == 'IssueTemplatesController') ||
       (context[:controller].class.name == 'GlobalIssueTemplatesController')
       o << javascript_include_tag('issue_templates', :plugin => 'redmine_issue_templates')
-    end      
-    return o
+    end
+    o
   end
   
   def view_issues_form_details_top(context={})
     action = context[:request].parameters[:action]
-    project_id = context[:request].parameters[:project_id]
-    issue_id =  context[:request].parameters[:id]
-
+    project = context[:project]
+    issue =  context[:issue]
+    setting = IssueTemplateSetting.find_or_create(project.id)
     copy_from = context[:request].parameters[:copy_from]
-    return '' if !copy_from.blank?
-    return '' unless (action == 'new' or action == 'update_form' or action == 'create') && !project_id.blank? && issue_id.blank?
+    is_triggered_by_status = is_triggered_by_status(context[:request])
+    return '' unless copy_from.blank?
+    return '' unless (action == 'new' or action == 'update_form' or action == 'create') && !project.id.blank? && issue.id.blank?
```

Comment:
__[rubocop]__
* Use `||` instead of `or`. [:ref](https://github.com/bbatsov/ruby-style-guide#no-and-or-or)
* Line is too long. [128/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)